seed/views/v3/uploads.py

```diff
@@ -46,7 +46,6 @@ class UploadViewSet(viewsets.ViewSet):
     @swagger_auto_schema(
         request_body=no_body,
         manual_parameters=[
-            AutoSchemaHelper.query_org_id_field(),
```

Comment:
I think we want to undo these changes and update the frontend to pass in an org_id for this request.